CommonTools/PileupAlgos/python/Puppi_cff.py

```diff
@@ -121,5 +123,30 @@
     puppi,
     EtaMinUseDeltaZ = 2.4,
     PtMaxCharged = 20.,
-    PtMaxNeutralsStartSlope = 20.
+    PtMaxNeutralsStartSlope = 20.,
+    NumOfPUVtxsForCharged = 2,
+    algos = cms.VPSet(
+                        cms.PSet(
+                         etaMin = cms.vdouble(-0.01), # include particles with eta==0 (proper fix is in #31174)
+                         etaMax = cms.vdouble(2.5),
+                         ptMin  = cms.vdouble(0.),
+                         MinNeutralPt   = cms.vdouble(0.2),
+                         MinNeutralPtSlope   = cms.vdouble(0.015),
+                         RMSEtaSF = cms.vdouble(1.0),
+                         MedEtaSF = cms.vdouble(1.0),
+                         EtaMaxExtrap = cms.double(2.0),
+                         puppiAlgos = puppiCentral
+                        ),
+                        cms.PSet(
+                         etaMin              = cms.vdouble( 2.5,  3.0),
+                         etaMax              = cms.vdouble( 3.0, 10.0),
+                         ptMin               = cms.vdouble( 0.0,  0.0),
+                         MinNeutralPt        = cms.vdouble( 1.7,  2.0),
+                         MinNeutralPtSlope   = cms.vdouble(0.08, 0.08),
+                         RMSEtaSF            = cms.vdouble(1.20, 0.95),
+                         MedEtaSF            = cms.vdouble(0.90, 0.75),
+                         EtaMaxExtrap        = cms.double( 2.0),
+                         puppiAlgos = puppiForward
+                        )
+     )
```

Comment:
IIUC,  only etaMin needs a change
```suggestion
     algos = { 0 : dict(etaMin = {-0.01}) } # include particles with eta==0 (proper fix is in #31174)
```